website/pages/docs/internals/security.mdx

```diff
@@ -0,0 +1,322 @@
+    and cause Nomad to avoid processing the request.
+
+* **Access to data without authentication or authorization** - Requests to the
+    server should be authenticated and authorized using mTLS and ACLs
+    respectively.
+
+* **State modification or corruption due to malicious messages** - Improperly
+    formatted messages are discarded while properly formatted messages require
+    authentication and authorization.
+
+* **Non-server members accessing raw data** - All servers that join the cluster
+    require proper authentication and authorization in order to begin
+    participating in Raft. All data in Raft should be encrypted with TLS.
+
+* **Denial of Service against a node** - DoS attacks against a single node
+    should not compromise the security posture of Nomad.
+
+The following are not part of the threat model for server agents:
+
+* **Access (read or write) to the Nomad data directory** - Information about the
+    jobs managed by Nomad is persisted to a server’s data directory.
+
+* **Access (read or write) to the Nomad configuration directory** - Access to
+    Nomad’s configuration file(s) directory can enable and disable features for
+    a cluster.
+
+* **Memory access to a running Nomad server agent** - Direct access to the
+    memory of the Nomad server agent process ( usually requiring a shell on the
+    system through various means ) results in almost all aspects of the agent
+    being compromised including access to certificates and other secrets.
+
+The following are not part of the threat model for client agents:
+
+* **Access (read or write) to the Nomad data directory** - Information about the
+    allocations scheduled to a Nomad client is persisted to its data directory.
+    This would include any secrets in any of the allocation’s file systems.
+
+* **Access (read or write) to the Nomad configuration directory** - Access to a
+    client’s configuration file can enable and disable features for a client
+    including insecure drivers such as
+    [raw_exec](/docs/drivers/raw_exec.html).
+
+* **Memory access to a running Nomad client agent** - Direct access to the
+    memory of the Nomad client agent process allows an attack to extract secrets
+    from clients such as Vault tokens.
+
+* **Lax Client Driver Sandbox** - Drivers may allow some privileged operations,
+    e.g. filesystem access to configuration directories, or raw accesses to host
+    devices. Such privileges can be used to facilitate compromise other workloads,
+    or cause denial-of-service attacks.
+
+#### Internal Threats
+
+* **Operator** - Someone with a valid mTLS cert and ACL token may still be a
+    threat to your cluster in certain situations, especially in multi-team
+    cluster deployments. They may accidentally or intentionally use a malicious
+    jobspec to harm a cluster which can help be protected against using
+    Namespaces and Sentinel policies.
```

Comment:
Quotas and Sentinel are probably the biggest thing to mention here as without Quotas any job operator can exhaust all cluster resources.